### ISSUE TITLE
Unexpected indentation warning

### ISSUE THREAD

**Issue by cchan** [sasstools/sass-lint#736]
### What version of Sass Lint are you using?
1.1.1

### What did you do? Please include the actual source code causing the issue.
```sass
body
  p
    font-size: 2.2em
```
### What did you expect to happen?
I was expecting it to validate. Am I doing something wrong?

### What actually happened? Please include any error messages given to you by Sass Lint.
```
[23:35:45] 
Starting 'sasslint'...

src\assets\styles\style.sass
  3:1  warning  Indentation of 0, expected 2  indentation

✖ 1 problem (0 errors, 1 warning)

[23:35:45] Finished 'sasslint' after 215 ms
```

**Comment by jackmcpickle** [comment]
I'm getting the same issue in atom. When linting sass files

running node 6.1.0 and sass-lint 1.7.0 - with [atom sass linter](https://github.com/AtomLinter/linter-sass-lint) 1.5.0

**Comment by DanPurdy** [comment]
There are many other open issues regarding this and a PR to fix this in #725.

#611 will explain the reasons why if you're interested.
